### DISCUSSION TITLE
Update information about Debian dependencies?

### DISCUSSION THREAD

**Discussion by wlbirula** [rbenv/rbenv#1577] [General]
When I tried to install a new version of ruby in Debian using rbenv, the error messages in the log file were quite laconic.
Wouldn't it be worthwhile to add dependencies to README.md?

For the default settings:

```sh
sudo apt install ruby-psych libffi-dev libyaml-dev 
```

**Comment by mislav** [comment]
Hi, we already link to our wiki under the "Installing Ruby versions" section of the README. Do you find that unclear, or that something was missing in the wiki instructions for Debian?
